staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go

```diff
@@ -426,6 +426,9 @@ type SAControllerConfiguration struct {
 	// serviceAccountKeyFile is the filename containing a PEM-encoded private RSA key
 	// used to sign service account tokens.
 	ServiceAccountKeyFile string
+	// concurrentServiceAccountSyncs is the number of service account syncing operations
+	// that will be done concurrently.
+	ConcurrentServiceAccountSyncs int32
```

Comment:
@liggitt Ugh, do we do names wrong (as it's done here) in literally every config type?

Reply:
do you mean not having json tags? this component doesn't yet read config from file, so we are free to rename fields, move them around, etc, prior to adding json tags.

that's on the roadmap to getting the components using external versioned config

Reply:
This is showing up as an API violation. I'd like to not let anything in that adds to the violation file. Reasonable or excessive request?

Reply:
that requires adding json tags to the file prior to actually thinking through what the serialized names should be. we didn't want to do that. a file with no json tags is clearly not intended to be serialized yet

Reply:
The tool is complaining that concurrentServiceAccountSyncs != ConcurrentServiceAccountSyncs; if the json tag were added, the tool would be happy. I honestly think the tool is kind of right.

Reply:
understood, but we did not want to add json tags before we intended the types to be serialized to files, because it implies they are serializable.